GNUmakefile

```diff
@@ -89,6 +92,9 @@ install-plugins: ## Build & install LOOPP binaries for products and chains.
 	go install $(GOFLAGS) ./pkg/solana/cmd/chainlink-solana
 	cd $(shell go mod download -json github.com/smartcontractkit/chainlink-starknet/relayer@$(STARKNET_SHA) | jq -r .Dir) && \
 	go install $(GOFLAGS) ./pkg/chainlink/cmd/chainlink-starknet
+	# Set GOPRIVATE to ensure go installs from our repository instead of the public proxy since we're using lightweight git tags.
+	cd $(shell GOPRIVATE=github.com/smartcontractkit/chainlink-internal-integrations go mod download -json github.com/smartcontractkit/chainlink-internal-integrations@$(APTOS_RELAYER_GIT_REF) | jq -r .Dir) && \
```

Comment:
It's not liking this go download/install path:

https://github.com/smartcontractkit/chainlink/actions/runs/14128740674/job/39583768935#step:4:1646

Reply:
Oh it should be the module, not the repo :facepalm: So need to move `aptos/relayer` from next line to here.

Reply:
Pushed: d4abab6c7e

Tested locally:

```shell
❯ cd $(go mod download -json github.com/smartcontractkit/chainlink-internal-integrations/aptos/relayer@2.21.0-beta16-aptos  | jq -r .Dir)
❯ go install ./cmd/chainlink-aptos
```

Reply:
Still seeing errors, will debug: https://github.com/smartcontractkit/chainlink/actions/runs/14131257297/job/39591975621#step:4:1309

Reply:
We're now green: https://github.com/smartcontractkit/chainlink/actions/runs/14135810191